tests/src/JIT/Performance/CodeQuality/BenchF/Adams/Adams.cs

```diff
@@ -91,16 +108,36 @@ public static void Test()
             }
         }
     }
+#endif
 
-    private static bool TestBase()
+    [MethodImpl(MethodImplOptions.NoOptimization)]
+    public static int Main(string[] argv)
     {
-        bool result = Bench();
-        return result;
-    }
+        if (argv.Length > 0)
```

Comment:
Some benchmarks recieve other arguments on the command line. So should this be `Adams -iter 5` instead of `Adams 5` to differentiate easily?